frontend/src/styles/global.scss

```diff
@@ -481,6 +473,9 @@ body {
     --link: var(--primary-3000);
     --tooltip-bg: var(--bg-charcoal);
     --data-color-1-hover: #1d4affe5;
+    --shadow-elevation: var(--shadow-elevation-3000);
+    --primary: var(--primary-3000);
+    --primary-highlight: var(--primary-3000-highlight);
```

Comment:
Oh nice! Didn't know it'd be that easy. 

That's actually where this PR came from. There's some incorrect color styles being applied in dark mode on the toolbar and the mix of `-3000` colors was confusing me so I thought it might be easier to strip some of this out first and see what was going on